web/cobrands/eastherts/_colours.scss

```diff
@@ -29,6 +25,8 @@ $col_click_map_dark: darken($eh_green, 10%);
 $col_fixed_label: #00BD08;
 $col_fixed_label_dark: #4B8304;
 
-$body-font: Verdana,Arial,Helvetica,Geneva,sans-serif;
+$body-font: 'Open Sans', sans-serif;
```

Comment:
I can't see where this font is being loaded, and things are appearing as `sans-serif` here.